src-java/kilda-model/src/main/java/org/openkilda/model/FlowPathDirection.java

```diff
@@ -0,0 +1,22 @@
+/* Copyright 2019 Telstra Open Source
+ *
+ *   Licensed under the Apache License, Version 2.0 (the "License");
+ *   you may not use this file except in compliance with the License.
+ *   You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   Unless required by applicable law or agreed to in writing, software
+ *   distributed under the License is distributed on an "AS IS" BASIS,
+ *   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *   See the License for the specific language governing permissions and
+ *   limitations under the License.
+ */
+
+package org.openkilda.model;
+
+public enum FlowPathDirection {
+    UNKNOWN,
```

Comment:
I can't explain why this value is possible. Up to now, we do not restrict the flow(also we do not have a separation between flow cookies and other cookies) cookie to have a specific direction. And I have no plans to add this restriction in this PR.

Reply:
Ok, then I'd call those cookies as with "unspecified" direction rather than UNKNOWN. The current version (UNKNOWN) sounds like we can't determine it for some reason...

Reply:
Renamed to "UNDEFINED".